### PR TITLE
Feature/cryoclim

### PR DESCRIPTION
**Pull request by trygveasp**

### CODE REVIEW

File: pysurfex/netcdf.py [hunk 1]
```diff
@@ -984,8 +984,6 @@ def read_cryoclim_nc(infiles):
             grid_snow_class_read = ncf["classed_product"][:]
```
Comment (josteinblyverket) [inline pysurfex/netcdf.py:984]: Depending on cryoclim files used "classed_product" is not always the variable we use.

File: pysurfex/cli.py [hunk 2]
```diff
@@ -1612,7 +1608,7 @@ def cryoclim_pseudoobs(argv=None):
         )
     else:
         logging.basicConfig(
-            format="%(asctime)s %(levelname)s %(message)s", level=logging.INFO
+            format="%(asctime)s %(levelname,)s %(message)s", level=logging.INFO
```
Comment (josteinblyverket) [inline pysurfex/cli.py:1611]: Bug? Removing "," seems to fix the KeyError: 'levelname,' error in CARRA

**Comment by trygveasp** [comment]
There are 2 PRs depending on this now, so we should merge this first as soon as it is accepted

**Comment by josteinblyverket** [comment]
Yes, I just want to check the -iv option in Cryoclim2json

**Review by josteinblyverket — APPROVED** [review]
Looks good to me. Some optimization needed for CARRA.

**Comment by trygveasp** [comment]
OK, then we merge this and do optimizations in another PR

**Pull request merged**